doc/source/examples/incompressible-flow/2d-backward-facing-step/2d-backward-facing-step.rst

```diff
@@ -255,126 +240,83 @@ For :math:`Re \geq 700`, however, it is often necessary to set ``ilu precondtion
 Running the Simulations
 -----------------------
 
-The simulation can be executed using the following command (assuming that the solver's location is in your PATH environment variable):
-
-.. code-block:: text
-  :class: copy-button
-
-  lethe-fluid 2D-backward-facing-step-steady.prm
-	
-However, mpi can be used to lower calculation time by using several CPUs (especially useful for pseudo-steady simulations) :
+The simulation can be executed using the following command (assuming that the solver's location is in your PATH environment variable and you want to use ``j`` processes for parallel computations):
 
 .. code-block:: text
   :class: copy-button
 
-  mpirun -np j lethe-fluid 2D-backward-facing-step-steady.prm
-	
-where ``j`` is the number of CPUs used for the computations.
+  mpirun -np j lethe-fluid Reynolds100_steady.prm
 
+For the case where :math:`\textrm{Re}=1000`, replace the name of the parameter file by ``Reynolds1000_steadybdf.prm``.
```

Comment:
same thing here, with or without the bfd

Reply:
nah I think this is ok right now. Because steadybdf is the name of the method